src/Core/Content/Media/MediaType/MediaType.php

```diff
@@ -8,19 +8,12 @@
 #[Package('discovery')]
 abstract class MediaType extends Struct
 {
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $name;
+    protected string $name;
 
     /**
      * @var array<string>
```

Comment:
Wouldn't `list<string>` better in that case?

Reply:
no because this would result in a list of string values but the flags property allows mixed indices

Reply:
an array key can only be a string or integer :wink:

with `array<string>` you already omit the information about the key type, so it can already be both